index.html

```diff
@@ -197,5 +197,22 @@ <h5>Skills</h5>
         </ul>
     </article>
  </section>
+
+ <footer class="contact">
+    <article>
+        <div class="C_Card">
+            <h3>Contact me</h3>
+            <p>If you have an application you are interested 
+                in developing, a feature that you need built or a 
+                project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/mnqwwwpj" method="POST">
+            <input type="text" name="name" placeholder="Enter your name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Enter your email" required>
+            <textarea name="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            <button type="submit" class="buttonBtn">Get It Touch</button>
+        </form>
+    </article>
+ </footer>
```

Comment:
- [ ] Kindly make sure that this section's background extends to the end of the page, to match the Figma design.
- [ ] Please use the same background color of the Figma template.
- [ ] Please add the rounded top left border to this section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/171225761-ed0e393e-25b0-463d-8220-ac02be442e1d.png) | ![image](https://user-images.githubusercontent.com/62290545/171225917-ed8864d3-61c4-4aac-8d53-d303462bf59d.png) |